common/prque/prque.go

```diff
@@ -28,24 +28,28 @@ type Prque struct {
 }
```

Comment:
How about updating the package descriptions on the top of the file?

Reply:
review reflected

Reply:
@winnie-byun Sorry for ambigious comment. I meant the following descriptions.
```
// Package prque implements a priority queue data structure supporting arbitrary
// value types and int64 priorities.	
//
// If you would like to use a min-priority queue, simply negate the priorities.
```
-> 
```
// Package prque implements a priority queue data structure supporting arbitrary
// value and priority types.
//
// If you would like to use a min-priority queue, simply put false as a reverse parameter.
```

Reply:
review reflected :)
I also change the word `reverse` to `invert`, since the word is more appropriate and also already used in go-eth.

Reply:
Thanks :)